lib/mpls.h

```diff
@@ -41,8 +41,10 @@
 #define MPLS_MAX_UNRESERVED_LABEL          1048575
 
 /* Default min and max SRGB label range */
-#define MPLS_DEFAULT_MIN_SRGB_LABEL        16000
-#define MPLS_DEFAULT_MAX_SRGB_LABEL        23999
+#define MPLS_DEFAULT_MIN_SRGB_LABEL        10000
```

Comment:
Let's add some comments here about the reasoning for these values choosen.  @odd22 and I had an offline discussion about this particular change and it seems like this is going to come up some more so let's leave a bread-crumb for future people.

Reply:
I would add that I'm not sure that it is safe to redistribute an OSPF-SR route in BGP. I mean redistribute an MPLS route with a specific label into BGP.